Resources/doc/cookbook/recipe_custom_action.rst

```diff
@@ -257,3 +257,7 @@ The full ``CarAdmin.php`` example looks like this:
             // ...
         }
     }
+
+.. note::
+
+    If you want to render a custom controller action in a template by using the render function in twig you need to add `_sonata_admin` as an attribute. For example; `{{ render(controller('AppBundle:XxxxCRUD:comment', { '_sonata_admin': 'sonata.admin.xxxx' })) }}`. This has to be done because the moment the rendering should happen no routing is involved yet, and then you will get an error 'There is no _sonata_admin defined for the controller AppBundle\Controller\XxxxCRUDController and the current route ''.'
```

Comment:
Ok, sorry. Some lack of experience I guess. Will fix this, and use branch 3.x instead.